trojsten/contests/templatetags/archive_parts.py

```diff
@@ -2,14 +2,18 @@
 
 from trojsten.contests.models import Competition
 
-from ..helpers import get_rounds_by_year
+from ..helpers import get_rounds_by_year, slice_if_needed
 
 register = template.Library()
 
 
 @register.inclusion_tag('trojsten/contests/parts/round_list.html')
 def show_round_list(user, competition):
     all_rounds = get_rounds_by_year(user, competition)
+
```

Comment:
Toto sa da ovela krajsie v pythone napisat:
Zadefinujes si 1 parametrovu fciu (mozes lokalne), napriklad `get_displayable_tags`, ktora vrati slice_if_needed(2, 1, param). Da sa pouzit aj functools.partial, ale to je myslim py3+
```
all_rounds = list(map(get_displayable_tags, all_rounds))
```

Reply:
popravde  
`for i in all_rounds:
   all_rounds[i] = [...]
`
som takto napisal lebo all_rounds je dict ktory vyrabal chyby ked som ho prepisal na list a co sa tyka toho list comprehensionu tak to by som mohol nejako skratit pomocou nejakeho mapu